minkasi/maps/mapset.py

```diff
@@ -230,6 +233,38 @@ def mpi_reduce(self):
             for map in self.maps:
                 map.mpi_reduce()
 
+class PriorMapset(Mapset):
+    """
+    Class to store and operate on WavSkyMaps. WavSkyMaps can be stored in Mapset but this
```

Comment:
Can you add more info on when you would store here vs `Mapset`? You subclass `Mapset` rather than `_MapsetBase` so this should be able to do everything `Mapset` does.

Reply:
Yeah documentation can be clearer. `PriorMapset` should be used to store `Mapset` which are used as priors on other `Mapset`s. Basically it just exposes a correct `apply_priors` for other functions/classes.